cylc/flow/cycling/iso8601.py

```diff
@@ -737,7 +737,15 @@ def prev_next(
     }
 
     for my_time in str_points:
-        parsed_point = parser.parse(my_time.strip())
+        try:
+            parsed_point = parser.parse(my_time.strip())
+        except ValueError:
+            suggest = my_time.replace(',', ';')
+            raise CylcConfigError(
```

Comment:
```suggestion
            raise WorkflowConfigError(
```
seems to be what is normally raised for this kind of thing